int16 x0
int16 y0
int16 image_x0
int16 image_y0
float32 theta_aruco